IdentityCore/src/network/request/MSIDAADAuthorizationCodeGrantRequest.m

```diff
@@ -44,6 +45,11 @@ - (instancetype)initWithEndpoint:(NSURL *)endpoint
         NSMutableDictionary *parameters = [_parameters mutableCopy];
         parameters[MSID_OAUTH2_CLIENT_INFO] = @YES;
         parameters[MSID_OAUTH2_CLAIMS] = claims;
+        if (enrollmentId != nil)
```

Comment:
it is redundant, the dictionary will not crash when you pass nil to subscript. It crashes only when you pass nit setObject:forKey.

Reply:
TIL - thank you 👍